packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -272,6 +272,32 @@ const fn = new lambda.Function(this, 'MyFunction', {
 See [the AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/concurrent-executions.html)
 managing concurrency.
 
+### Configuring AutoScaling
+
```

Comment:
Add some verbiage from the AWS documentation which describes this feature (usually the first/second paragraph of the doc)